uvicorn/protocols/http/httptools_impl.py

```diff
@@ -237,6 +241,12 @@ def on_header(self, name: bytes, value: bytes) -> None:
         if name == b"expect" and value.lower() == b"100-continue":
             self.expect_100_continue = True
         self.headers.append((name, value))
+        if self.config.limit_request_header_size and (
```

Comment:
I wanted to check lengths of name and values individually. Other implementations do the same.